packages/clients-app/src/root.component.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { render } from "@testing-library/react";
+import Root from "./root.component";
+
+describe("Root component", () => {
+  // it("should be in the document", () => {
+  //   const { getByText } = render(<Root name="Testapp" />);
+  //   expect(getByText(/Testapp is mounted!/i)).toBeInTheDocument();
+  // });
+});
```

Comment:
Consider removing this for now till we start implementing test.

Reply:
When running the npm test command with the --passWithNoTests flag it still fails. So I will just leave this place holder to prevent failures.

Reply:
It will still fail. So it's best to delete the file all together